src/Distortion.cpp

```diff
@@ -200,6 +215,62 @@ void removeDistortion(double dx, double dy, double &ux, double &uy,
       ux = dx + dr_x;
       uy = dy + dr_y;
     }
+    break;
+    // The dawn distortion model is "reversed" so the remove function computes
```

Comment:
I understand the situation, but found this comment confusing. The `DAWNFC` case under `removeDistortion` uses an iterative solution to calculate the undistorted coordinates, given the distorted coordinates. I wouldn't say that it it computes one thing but stores it in the other's variables? 

The iterative solution is necessary here because Dawn's distortion equation is given as distorted = fn(undistorted), which is reversed from the more normal undistorted = fn(distorted).